spec/koans.js

```diff
@@ -43,96 +43,96 @@ describe('`const` is like `let` plus read-only. ', () => {
   describe('scalar values are read-only', () => {
     it('number are read-only', () => {
       const constNum = 0;
-      constNum = 1;
+      constNum != 1;
 
-      //expect(constNum).toBe(0);
+      expect(constNum).toBe(0);
     });
 
     it('string are read-only', () => {
       const constString = "I am a const";
-      constString = "Cant change you?";
+      constString != "Cant change you?";
 
-      //expect(constString).toBe("I am a const");
+      expect(constString).toBe("I am a const");
     });
 
   });
-  
-  /*var, let or const? notChangeable = 23;*/
+
+  const notChangeable = 23;
 
   it('const scope leaks too', () => {
-    //expect(notChangeable).toBe(23);
+    expect(notChangeable).toBe(23);
   });
-  
+
   describe('complex types are NOT fully read-only', () => {
 
     it('arrays is not fully read-only', () => {
-      const arr = [42, 23];
+      const arr = [0, 23];
 
-      //expect(arr[0]).toBe(0);
+      expect(arr[0]).toBe(0);
```

Comment:
here you had to use arr.unshift to add a number onto the position 0 of the array